src/test/java/com/exasol/github/JavaMavenProjectTest.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.github;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHContent;
+import org.kohsuke.github.GHRepository;
+import org.mockito.Mockito;
+
+class JavaMavenProjectTest {
+    @Test
+    void testGetVersionWithCaching() throws IOException {
+        final GHRepository ghRepositoryMock = Mockito.mock(GHRepository.class);
+        final GHContent contentMock = Mockito.mock(GHContent.class);
+        final String version = "<project><version>1.0.0</version></project>";
```

Comment:
Try different variants with spaces and newlines in that field. While that is bad practice, it is valid XML.

Reply:
done